oak_attestation_verification/src/verifier.rs

```diff
@@ -517,14 +517,27 @@ fn verify_kernel_layer(
     )
     .context("kernel failed verification")?;
 
-    verify_regex(
-        values.kernel_raw_cmd_line.as_str(),
-        reference_values
-            .kernel_cmd_line_regex
-            .as_ref()
-            .context("no kernel command line regex reference values")?,
-    )
-    .context("kernel command line failed verification")?;
+    if let Some(kernel_raw_cmd_line) = values.kernel_raw_cmd_line.as_ref() {
+        verify_regex(
+            kernel_raw_cmd_line.as_str(),
+            reference_values
+                .kernel_cmd_line_regex
+                .as_ref()
+                .context("no kernel command line regex reference values")?,
+        )
+        .context("kernel command line failed verification")?;
+    } else {
+        // Support missing kernel_cmd_line_regex but only if the corresponding reference
+        // value is set to skip. This is a temporary workaround until all clients are
+        // migrated.
+        anyhow::ensure!(
+            matches!(
+                reference_values.kernel_cmd_line_regex.as_ref().unwrap().r#type.as_ref(),
```

Comment:
I also just noticed that this `unwrap()` call will crash/panic if no `kernel_cmd_line_regex` field is specified in the ReferenceValues, as opposed to generating a (recoverable) error., which seems undesirable given that that field is now optional.